### PR TITLE
[Teams] Fix onSelectedItem in Teams Search Extension Bot

### PR DESCRIPTION
**Pull request by ceciliaavila**
Addresses # 1314

## Description
This PR fixes how the [teamsSearchExtensionBot](https://github.com/microsoft/botbuilder-js/blob/master/libraries/teams-scenarios/searchBasedMessagingExtension/src/teamsSearchExtensionBot.ts) handles the _onSelectedItem_ event so it returns a hero card for every result selected by the user.

### Detailed changes
- Updated [teamsSearchExtensionBot.ts](https://github.com/microsoft/botbuilder-js/blob/master/libraries/teams-scenarios/searchBasedMessagingExtension/src/teamsSearchExtensionBot.ts) file adding a tap event to all the result card' previews.

## Testing
The images below show the bot running on Teams.

image

### CODE REVIEW

File: libraries/teams-scenarios/searchBasedMessagingExtension/src/index.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import * as restify from 'restify';
 
 // Import required bot services.
 // See https://aka.ms/bot-services to learn more about the different parts of a bot.
-import { BotFrameworkAdapter, MemoryStorage, UserState } from 'botbuilder';
+import { BotFrameworkAdapter, MemoryStorage, UserState } from '../../../botbuilder';
```
Comment (sw-joelmut) [inline libraries/teams-scenarios/searchBasedMessagingExtension/src/index.ts:10]: Do we need this change? from importing `botbuilder` to `../../../botbuilder`

File: libraries/teams-scenarios/searchBasedMessagingExtension/src/teamsSearchExtensionBot.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ import {
     MessagingExtensionResult,
     TeamsActivityHandler,
     TurnContext
-} from 'botbuilder';
+} from '../../../botbuilder';
```
Comment (sw-joelmut) [inline libraries/teams-scenarios/searchBasedMessagingExtension/src/teamsSearchExtensionBot.ts:16]: Do we need this change? from importing `botbuilder` to `../../../botbuilder`

File: libraries/teams-scenarios/searchBasedMessagingExtension/src/index.ts (hunk 1 above)
Comment (ceciliaavila) [reply libraries/teams-scenarios/searchBasedMessagingExtension/src/index.ts]: I reverted it. Thanks!

File: libraries/teams-scenarios/searchBasedMessagingExtension/src/teamsSearchExtensionBot.ts (hunk 2 above)
Comment (ceciliaavila) [reply libraries/teams-scenarios/searchBasedMessagingExtension/src/teamsSearchExtensionBot.ts]: I reverted it. Thanks!

**Pull request closed without merging**